FunctionsHW8.cpp

```diff
@@ -0,0 +1,89 @@
+#include "FunctionsHW8.h"
+
+int factorialRec(unsigned int count) {
+	return (count == 0) ? 1 : count * factorialRec(count - 1);
```

Comment:
count == 1

Reply:
And if we put 0 as a parameter of factorialRec?